spotbugs/src/test/java/edu/umd/cs/findbugs/util/MutableClassesTest.java

```diff
@@ -14,6 +30,21 @@ public void TestKnownImmutable() {
         Assert.assertFalse(MutableClasses.mutableSignature("Ljava/lang/String;"));
     }
 
+    @Test
+    public void TestKnownImmutablePackage() {
+        Assert.assertFalse(MutableClasses.mutableSignature("Ljava/time/LocalTime;"));
+    }
+
+    @Test
+    public void TestNamedImmutable() {
```

Comment:
ImmutableMap appears to be explicitly defined, not relying on name.

Reply:
This conversation can be resolved, since this testcase got removed in https://github.com/spotbugs/spotbugs/pull/1619/commits/65a43889a16ec6fc0f4c919e4cd4710a0e8d4966, since the checker doesn't check for the string "immutable" in the name.